internal/router/controllers/user.go

```diff
@@ -331,3 +351,17 @@ func getFormatterParams(r *http.Request) (formatterParams, error) {
 
 	return c, nil
 }
+
```

Comment:
I'm not sure if I'm liking this design choice of having to provide an implementation of an interface defined in the telemetry package. Kind of a pain to have to define a new struct just for that. If we could define an anonymous implementation of an interface it would be better. Just mentioning if other ideas come to mind.

Reply:
Yes, it felt somewhat heavy the last time I added a metric. We can meditate around changing this.
Worth a GH issue to track?

Reply:
We could 👍